src/components/sidebar.js

```diff
@@ -19,46 +19,43 @@ const Sidebar = (props) => {
             <div className="close">
               <img src={closeIcon} alt="" onClick={props.close}></img>
             </div>
-            <div className="link" ref={(e) => (headingsRef.current[0] = e)}>
-              <div className="heading" onClick={() => onCollapseClick(0)}>
+            <ul className="link" ref={(e) => (headingsRef.current[0] = e)}>
+              <li className="heading" onClick={() => onCollapseClick(0)}>
                 <a href="/">
                   <h2>Company</h2>
                 </a>
                 <img src={arrowIcon} alt=""></img>
-              </div>
-              <h3>About</h3>
-              <h3>Team</h3>
-              <h3>Press</h3>
-            </div>
-            <div className="divisor"></div>
-            <div className="link">
-              <a href="/services">
-                <h2>Services</h2>
-              </a>
-            </div>
-            <div className="divisor"></div>
-            <div className="link" ref={(e) => (headingsRef.current[1] = e)}>
-              <div className="heading" onClick={() => onCollapseClick(1)}>
+              </li>
+              <li>About</li>
+              <li>Team</li>
+              <li>Press</li>
+            </ul>
+            <ul className="link">
+              <li className="heading">
+                <a href="/services">
+                  <h2>Services</h2>
+                </a>
+              </li>
+            </ul>
+            <ul className="link" ref={(e) => (headingsRef.current[1] = e)}>
+              <li className="heading" onClick={() => onCollapseClick(1)}>
                 <a href="/">
                   <h2>Our work</h2>
                 </a>
                 <img src={arrowIcon} alt=""></img>
```

Comment:
Instead of adding  ` <img src={arrowIcon} alt="" />`  to each top-level link, you can add it to the `heading` class using CSS so you don't manually add the image to all heading links.

Here's an example mixin from a different project that adds a plus icon svg as a background image.
```
@mixin icon-plus-alt {
  background:
          no-repeat
          right
          url('data:image/svg+xml; utf8, <svg xmlns="http://www.w3.org/2000/svg" width="401.994" height="401.994" viewBox="0 0 401.994 401.994"><path fill="%235B616B" d="M394 154.175c-5.33-5.33-11.806-7.994-19.417-7.994H255.81V27.41c0-7.61-2.665-14.084-7.993-19.414C242.487 2.666 236.02 0 228.397 0h-54.81c-7.613 0-14.085 2.663-19.415 7.993-5.33 5.33-7.994 11.803-7.994 19.414v118.775H27.408c-7.612 0-14.085 2.664-19.415 7.994S0 165.973 0 173.59v54.818c0 7.618 2.662 14.086 7.992 19.41 5.33 5.333 11.803 7.995 19.414 7.995h118.77V374.59c0 7.61 2.665 14.09 7.995 19.417 5.33 5.325 11.806 7.987 19.418 7.987H228.4c7.618 0 14.087-2.662 19.418-7.987 5.332-5.33 7.994-11.806 7.994-19.417V255.813h118.77c7.618 0 14.09-2.662 19.417-7.994 5.325-5.33 7.99-11.797 7.99-19.415v-54.82c0-7.615-2.66-14.086-7.993-19.413z"/></svg>');
  background-size: calculate-rem(10px);
}
```

If you upload the arrow icon into this site https://jakearchibald.github.io/svgomg/ it will generate stripped down markup that you can place in the `url('data:image` section. 

**Important**: Don't forget to change any hex colors eg.`#fff` to `%23fff`. If you don't change the hash sign to %23, svg icons will not get displayed in Firefox.

To add the icon to the top-level heading, you can use the `::after` psuedo-selector
An eg from a different project:

```
.heading::after {
      @include icon-plus-alt;
      background-size: calculate-rem(12px);
      display: inline-block;
      content: '';
      width: 25px;
      height: 18px;
      vertical-align: sub;
    }
```